handbook/company/pricing-features-table.yml

```diff
@@ -767,15 +751,6 @@
   productCategories: [Device management]
   pricingTableCategories: [Device management]
   moreInfoUrl: https://github.com/fleetdm/fleet/issues/18867
-- industryName: Puppet module
```

Comment:
Removed because it is not an MDM feature and is duplicative of “Premium integrations (Puppet, Vanta, Jira, Zendesk, etc)” from the integrations section.